pricing/v1/pricing.proto

```diff
@@ -46,23 +46,34 @@ message GetInfoResponse {
 }
 
 // Request message for GetPricingInfo
-message GetPricingInfoRequest {
+message GetPricingRequest {
+
+  //Required cloud (AWS/Azure)
   string cloud = 1;
+
+  // Required service. 
+  //EC2, Lambda, S3, Elastic Block Storage, Data Transfer, RDS, DynamoDB, and Cloud Watch for AWS
+  //Virual Machine, Functions, Blob Storage, Storage Account, Data Transfer, SQL Database, Cosmos DB, and Monitor for Azure
   string service = 2;
+
+  // Required region.
   string region = 3;
+
+  // Required token for continuing request
   string token = 4;
+  // Optional column number
```

Comment:
This array is actually an array of strings that represent valid columns which will serve as basis as to what columns will be returned by the API. Not just the number of columns to be returned. Also state that all columns will be returned by default if this array is empty. Please note in the description that this is not implemented yet.